### PR TITLE
fix is empty

### PR DESCRIPTION
**Pull request by davgur**

### CODE REVIEW

File: src/helpers/__tests__/utils.test.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
+import { isEmpty } from '../utils';
+
+describe('isEmpty', () => {
+  test('object', () => {
```
Comment (bbfsdev) [inline src/helpers/__tests__/utils.test.js:4]: Add null/undefined cases.

File: src/helpers/__tests__/utils.test.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    expect(isEmpty('aaa')).toBe(false);
+    const a = new String();
+    expect(isEmpty(a)).toBe(true);
+    expect(isEmpty('')).toBe(true);
+
+  });
+
+  test('symbol', () => {
+    const a = Symbol();
+    expect(isEmpty(a)).toBe(false);
```
Comment (bbfsdev) [inline src/helpers/__tests__/utils.test.js:81]: Empty symbol is not empty?

File: src/helpers/utils.js [hunk 3]
```diff
@@ -34,24 +35,19 @@ export const isEmpty = obj => {
...
+    return Object.getOwnPropertyNames(obj).length === 0;
   }
 
-  // If it isn't an object at this point
-  // it is empty, but it can't be anything *but* empty
-  // Is it empty?  Depends on your application.
-  if (typeof obj !== 'object') {
-    return true;
+  if (typeof obj == 'number') {
+    return isNaN(obj);
```
Comment (bbfsdev) [inline src/helpers/utils.js:47]: You have not testing this.

Why   !isNaN    should be not empty?

File: src/helpers/__tests__/utils.test.js [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+      const a = [1];
+      expect(isEmpty(a[0])).toBe(false);
+      expect(isEmpty(a[1])).toBe(true);
+      a.pop();
+      expect(isEmpty(a[0])).toBe(true);
+    }
+  });
+
+  test('number', () => {
+    expect(isEmpty(1)).toBe(false);
```
Comment (bbfsdev) [inline src/helpers/__tests__/utils.test.js:66]: I would guess any number is empty... 

Why !isNaN is not empty?
What about 0?

File: src/helpers/utils.js (hunk 3 above)
Comment (davgur) [reply src/helpers/utils.js:47]: a blin, you right

Comment (davgur) [reply src/helpers/utils.js:47]: when its NaN its empty (isNaN(obj) === true) thats its return

File: src/helpers/__tests__/utils.test.js (hunk 4 above)
Comment (davgur) [reply src/helpers/__tests__/utils.test.js:66]: you think that need add 0 as empty?

**Review by bbfsdev — APPROVED** [review]

**Pull request merged**